src/ray/common/memory_monitor.cc

```diff
@@ -123,29 +123,26 @@ int64_t MemoryMonitor::GetCGroupV1MemoryUsedBytes(const char *path) {
   int64_t value;
 
   int64_t rss_bytes = kNull;
-  int64_t cache_bytes = kNull;
   int64_t inactive_file_bytes = kNull;
   while (std::getline(memstat_ifs, line)) {
     std::istringstream iss(line);
     iss >> title >> value;
     if (title == "total_rss") {
       rss_bytes = value;
-    } else if (title == "total_cache") {
-      cache_bytes = value;
     } else if (title == "total_inactive_file") {
```

Comment:
`total_cache` item in `/sys/fs/cgroup/memory/memory.stat` should means "Linux OS page cache / buffer", this part memory should be count as "available", because this part memory is borrowed by Linux OS to cache Page or buffers but if user process want to allocate more memory , Linux will evict them immediately and allocate to user process.